spec/script/setup_concepts_spec.rb

```diff
@@ -0,0 +1,96 @@
+require 'spec_helper'
+require 'rails_helper'
+require 'setup_concepts'
+
+VCR.configure do |config|
+  config.cassette_library_dir = "fixtures/vcr_cassettes"
+  config.hook_into :webmock # or :fakeweb
+end
+
+describe 'Setting up concepts' do
+  VCR.use_cassette('fetching concepts') do
+
+    setup = Setup::Concepts.new
+    
+    it "should have an attribute concepts" do
+      expect(setup.concepts.class).to eq(Array)
+    end
+
+    response = setup.fetch_concepts()
+
+    it "should successfully make a http request" do
+      expect(response.code).to eq("200")
+    end
+
+    it "the response should be a string" do
+      expect(response.body.class).to eq(String)
+    end
+
+    it "can convert the string response to JSON" do
+      expect(setup.string_to_json(response.body).class).to eq(Hash)
+    end
+
+    it "hash of data has a list of concepts" do
+      expect(setup.string_to_json(response.body)["concepts"].class).to eq(Array)
+    end
+
+    it "has a hash that represents a concepts inside the list" do
+      expect(setup.concepts.first.class).to eq(Hash)
+    end
+
+    it "stores the data" do
+      expect(setup.concepts.empty?).to eq(false)
+    end
+
+    it "should have no concepts in the db at runtime" do
```

Comment:
this tests the test-setup, so I don't think it should be in the specs.
even better, perhaps we can make a test that is agnostic to the state of the db